src/pages/index.astro

```diff
@@ -3,5 +3,78 @@ import Layout from "../layouts/Layout.astro";
 ---
 
 <Layout>
-  <h1 class="font-sans text-4xl">nunito</h1>
+  
+  <div class="box">
+    <div class="box1">
+      <h4 class="title">NATIONAL PENSION SCHEME</h4>
+      <h2 class="text-black">A happier</h2>
+      <h2 class="text-purple">tomorrow.</h2>
+      <h2 class="text-black">A happier</h2>
+      <h2 class="text-purple">today</h2>
+      <p>Invest in the National Scheme to save <br /> for your retirement and save on tax today</p>
+      <h3 class="tag"><i class="fa-solid fa-lock"></i>NPS POP REG NO 1614858</h3>
+    </div>
+    <div class="box2">
+      <img class="img" src="/public/img.jpg" alt="">
+    </div>
+  </div>
```

Comment:
Please create a new component called `Banner.astro` instead of putting all your markup and styles here. Your component should be located `/src/components`.